src/plugin.ts

```diff
@@ -76,8 +80,8 @@ function activateWidgetExtension(app: JupyterLab, registry: IJupyterWidgetRegist
       name: '@jupyter-widgets/jupyterlab-sidecar',
       version: EXTENSION_SPEC_VERSION,
       exports: {
-        SidecarModel: SidecarModel,
-        SidecarView: SidecarView
+        // @ts-ignore: Why is this not compiling?
```

Comment:
Shamelessly ignoring this error... As I really don't know how to fix it.

The error message is:
```
$ tsc
src/plugin.ts:83:9 - error TS2322: Type 'typeof SidecarModel' is not assignable to type 'typeof WidgetModel | typeof WidgetView'.
  Type 'typeof SidecarModel' is not assignable to type 'typeof WidgetModel'.
    Construct signature return types 'SidecarModel' and 'WidgetModel' are incompatible.
      The types of 'widget_manager.display_view' are incompatible between these types.
        Type '(msg: any, view: View<Model<any, ModelSetOptions, {}>>, options: any) => Promise<Widget>' is not assignable to type '(msg: IMessage<MessageType>, view: DOMWidgetView, options: any) => Promise<any>'.
          Types of parameters 'view' and 'view' are incompatible.
            Type 'DOMWidgetView' is not assignable to type 'View<Model<any, ModelSetOptions, {}>>'.
              Types of property 'preinitialize' are incompatible.
                Type '(options?: ViewOptions<WidgetModel>) => void' is not assignable to type '(options?: ViewOptions<Model<any, ModelSetOptions, {}>>) => void'.
                  Types of parameters 'options' and 'options' are incompatible.
                    Type 'ViewOptions<Model<any, ModelSetOptions, {}>>' is not assignable to type 'ViewOptions<WidgetModel>'.
                      Type 'Model<any, ModelSetOptions, {}>' is missing the following properties from type 'WidgetModel': comm_live, send, close, _handle_comm_closed, and 23 more.

83         SidecarModel, SidecarView
           ~~~~~~~~~~~~

src/plugin.ts:83:23 - error TS2322: Type 'typeof SidecarView' is not assignable to type 'typeof WidgetModel | typeof WidgetView'.
  Type 'typeof SidecarView' is missing the following properties from type 'typeof WidgetModel': _deserialize_state, serializers

83         SidecarModel, SidecarView
                         ~~~~~~~~~~~


Found 2 errors.
```

Which doesn't really makes sense because the SidecarModel inherits from WidgetModel

Reply:
This may happen, for example, if there is a version mismatch or typo somewhere?